### ISSUE TITLE
Jim unable to connect to GLCWRA Jump server

### ISSUE THREAD

**Issue by lprivette** [USGS-WiM/GBRA#8]
Can we please reset his password? 

![image](https://user-images.githubusercontent.com/23640741/57033239-64f15d80-6c1a-11e9-8ae3-ed8df88f6ba1.png)

**Comment by aaronstephenson** [comment]
@DanielJBeckman can you take this one?

**Comment by lprivette** [comment]
@DanielJBeckman any chance we could do this today?

**Comment by aaronstephenson** [comment]
I'll do it now since Dan is having an IT snafu.

**Comment by aaronstephenson** [comment]
Done.

**Comment by DanielJBeckman** [comment]
So this is alive again. Jim is not in AD so I am guessing this is a AWS issue.

**Comment by aaronstephenson** [comment]
@lprivette Can you give me more information about what's happening now?

**Comment by lprivette** [comment]
I reset his password and tested that I was able to login with his username and the new password. 

Despite my successful login, he's still unable to login with those same creds. 

I asked him to verify that he did not have the "GS" domain before his username and he said it was not there.

So I emailed him last week asking if we could hop on a hangout so that I could actually see what he's seeing. There's been no response. He doesn't seem to want to troubleshoot it.

**Comment by aaronstephenson** [comment]
He could try using the computer name as the domain, since he's using a local account:

GISJMP105GLCWRA

**Comment by lprivette** [comment]
Blake, Hans and I walked through it with him and something is definitely wrong. @aaronstephenson 

I'm still able to login as him. We compared our settings but he's still unable to. 

We tried with Remote Manager and standard remote desktop but the message error was still the same "Login attempt failed".

We were going to try VPNing to see if it maybe is an issue with the IP ranges, but he doesn't have pulse on his desktop.

**Comment by aaronstephenson** [comment]
So, the account is valid and not locked and we have confirmed that it works when someone else attempts to log in with it. The only time it doesn't work is when Jim himself attempts to log in.

I have also confirmed that RDP is allowed to all of the Middleton office's subnets, and all the USGS VPN subnets, unless those have changed in the last few months since I last saw a list. What is is IP address?

This sounds to me like it's either a problem with his desktop or his local network, neither of which I can troubleshoot, but the Middleton IT staff can.
